src/components/cms/SkillFeedbackCard/SkillFeedbackCard.js

```diff
@@ -224,6 +254,18 @@ class SkillFeedbackCard extends Component {
                     : `${userEmail.slice(0, userEmail.indexOf('@') + 1)}...`}
                 </div>
                 <Timestamp>{formatDate(parseDate(data.timestamp))}</Timestamp>
+                <Ratings
+                  rating={3}
```

Comment:
Just need to replace `rating` with `data.stars`. It's hardcoded to 3 for now

Reply:
Do this change here are edit the PR description with , 
TO BE MERGED AFTER #SERVER_PR

Reply:
If I do that change now the checks will fail and it will not get deployed to surge.